core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2171,7 +2171,7 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_REPLICATION_CHECK_INTERVAL_MS =
       new Builder(Name.MASTER_REPLICATION_CHECK_INTERVAL_MS)
-          .setDefaultValue("1min")
+          .setDefaultValue("10sec")
```

Comment:
any particular reason why we want to update this?

Reply:
Yeah. The reason why this number had to be so high before was because it is scheduling all of the jobs at once until it's full. Now that it doesn't have to be that, it can run much more frequently.

Reply:
each check has non trivial amount of overhead (checking all pinned files) and can be quite large if the number of pinned files is large. can we leave this at 1min?

Reply:
This is going to have a side effect of replication taking a much longer time than before for a large number of files. That is okay?

Reply:
In the worst case, this will end up equating to being about 10x slower since we are using at most 10% of the job service capacity. The other thing here is that if job service is at capacity for other reasons, our job retention time for finished jobs is 60 seconds so this needs to be less than 60 seconds if you are writing code that looks for job status. I wrote code such that it assumes it has finished if the job status is unavailable but it shouldn't get there by default.